engine/flink/kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/kafka/consumerrecord/ConsumerRecordDeserializationSpec.scala

```diff
@@ -23,8 +28,10 @@ class ConsumerRecordDeserializationSpec extends AnyFunSuite with Matchers with K
     val typeInformation: TypeInformation[TestConsumerRecord] = TypeInformation.of(classOf[TestConsumerRecord])
 
     intercept[Exception] {
+      Serializers.registerSerializers(emptyModel, executionConfigWithoutKryo)
```

Comment:
this invocation could be moved outside of `intercept`? so we are 100% sure that exception is thrown by `getSerializeRoundTrip`

Reply:
:+1: